registries/_format/base64url.md

```diff
@@ -0,0 +1,35 @@
+---
+owner: baywet
+issue:
+description: A url-safe binary array as defined in RFC4648
+base_type: string
+layout: default
+oas_version: < 3.1
+remarks: "When using OpenAPI 3.1 or above it's recommended not to use this format and instead use [`contentEncoding` with a value of `base64url`](https://json-schema.org/draft/2020-12/json-schema-validation.html#name-contentencoding)."
```

Comment:
Cool! Can we have similar text in `byte.md`, with "base64" instead of "base64url"?

Can't propose it there because it's in the "old" part of the file.

Reply:
waiting for the resolution of this thread before we do so.
https://github.com/OAI/OpenAPI-Specification/pull/3167#discussion_r1113067249
And it looks like what we've just added to base64url might need some rewording if we decide using it for properties of the payload is ok in 3.1